tests/validation/tests/v3_api/test_workload.py

```diff
@@ -42,7 +42,7 @@ def test_wl_sidekick():
     validate_workload_with_sidekicks(
         p_client, workload, "deployment", ns.name)
 
-
+#Converted to go test in TestWorkloadDeployment
```

Comment:
I dont think we will have to add this comment for each of the tests. we can later deprecate these tests.

Reply:
That is true, but there have been instances in the past where tests have been converted but not documented, so this is a low-tech solution to avoid such circumstances.